.ci/run_tests.sh

```diff
@@ -129,7 +125,7 @@ mkdir -p /tmp/minio_tests/test-bucket && chmod -R 777 /tmp/minio_tests
 # Create an empty webdav folder with appropriate permissions so www user can write inside it
 mkdir -p /tmp/webdav_tests && chmod 777 /tmp/webdav_tests
 
-docker-compose \
+docker compose \
```

Comment:
Why this change ? Is related to QWT at all ? It broke runs with podman, see GH-57724

Reply:
At the time GH dropped docker compose v1 (before bring it back after users complained).
They are planning to drop it on July, 9 actions/runner-images#9692

Reply:
I don't think GH runs .ci/run_tests.sh though, that script is for humans usage